Travel-Planner/Travel-Planner.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,13 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		83DE71392CFFF8880010BE8B /* DateRangeSelection.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71362CFFF8880010BE8B /* DateRangeSelection.swift */; };
+		83DE713A2CFFF8880010BE8B /* DateCellView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71352CFFF8880010BE8B /* DateCellView.swift */; };
+		83DE713B2CFFF8880010BE8B /* CustomCalendarView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71342CFFF8880010BE8B /* CustomCalendarView.swift */; };
+		83DE713E2CFFF88C0010BE8B /* TripCardView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE713C2CFFF88C0010BE8B /* TripCardView.swift */; };
+		83DE71442D0033650010BE8B /* ImageFileManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71432D0033650010BE8B /* ImageFileManager.swift */; };
+		83DE71462D0034840010BE8B /* ActivityTypePickerView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71452D0034840010BE8B /* ActivityTypePickerView.swift */; };
+		83DE717A2D02C9A20010BE8B /* DeleteMode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83DE71792D02C9A20010BE8B /* DeleteMode.swift */; };
```

Comment:
There no DeleteMode.swift in my project. Some how it appear and cause a run error. How to fix it.